### PR TITLE
Lehmer Generator

### PR DESCRIPTION
**Pull request by Lin-Rin**

### CODE REVIEW

File: src/main/LehmerGenerator.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    private static final int M = (int) Math.pow(2, 32);
+    private static final int A = (int) Math.pow(2, 16) + 1;
+    private static final int C = 119;
+    private int seed;
+
+    public LehmerGenerator() {
+        seed = new Random().nextInt();
+    }
+
+    public LehmerGenerator(int seed) {
```
Comment (kosssst) [inline src/main/LehmerGenerator.java:15]: add exception if 0

File: src/main/LehmerGenerator.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        this.seed = seed;
+    }
+
+    public int lehmerHigh() {
```
Comment (kosssst) [inline src/main/LehmerGenerator.java:19]: rework

**Review by kosssst — CHANGES_REQUESTED** [review]

File: src/main/Main.java [hunk 3]
```diff
@@ -2,6 +2,6 @@
 
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
```
Comment (kosssst) [inline src/main/Main.java:5]: place it back

**Review by kosssst — CHANGES_REQUESTED** [review]

File: src/main/Main.java (hunk 3 above)
Comment (Lin-Rin) [reply src/main/Main.java:5]: ???

**Review by kosssst — APPROVED** [review]

**Pull request merged**